tasks/system_probe.py

```diff
@@ -357,16 +357,16 @@ def ninja_test_ebpf_programs(nw: NinjaWriter, build_dir):
 
 
 def ninja_gpu_ebpf_programs(nw: NinjaWriter, co_re_build_dir: Path | str):
-    gpu_programs_co_re_dir = Path("pkg/gpu/ebpf/c")
-    gpu_programs_co_re_flags = f"-I{gpu_programs_co_re_dir}"
-    gpu_programs_co_re_programs = ["gpu"]
+    gpu_c_dir = Path("pkg/gpu/ebpf/c")
+    gpu_flags = f"-I{gpu_c_dir} -g"
+    gpu_programs = ["runtime/gpu"]
```

Comment:
I think this is making the build code place the object in the wrong folder, that's why the KMT tests are all red. Something like this 
```suggestion
    gpu_inc_dir = Path("pkg/gpu/ebpf/c")
    gpu_c_dir = gpu_inc_dir / "runtime"
    gpu_flags = f"-I{gpu_inc_dir} -g"
    gpu_programs = ["gpu"]
```